docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -1,62 +1,64 @@
 ---
-title: Use HttpClientFactory to implement resilient HTTP requests
-description: Learn how to use HttpClientFactory, available since .NET Core 2.1, for creating `HttpClient` instances, making it easy for you to use it in your applications. 
+title: Use IHttpClientFactory to implement resilient HTTP requests
+description: Learn how to use IHttpClientFactory, available since .NET Core 2.1, for creating `HttpClient` instances, making it easy for you to use it in your applications. 
 ms.date: 08/08/2019
 ---
-# Use HttpClientFactory to implement resilient HTTP requests
+# Use IHttpClientFactory to implement resilient HTTP requests
 
-`HttpClientFactory` is an opinionated factory, available since .NET Core 2.1, for creating <xref:System.Net.Http.HttpClient> instances to be used in your applications.
+<xref:System.Net.Http.IHttpClientFactory> is a contract implemented by `DefaultHttpClientFactory`, an opinionated factory, available since .NET Core 2.1, for creating <xref:System.Net.Http.HttpClient> instances to be used in your applications.
 
 ## Issues with the original HttpClient class available in .NET Core
 
 The original and well-known <xref:System.Net.Http.HttpClient> class can be easily used, but in some cases, it isn't being properly used by many developers.
 
-As a first issue, while this class is disposable, using it with the `using` statement is not the best choice because even when you dispose `HttpClient` object, the underlying socket is not immediately released and can cause a serious issue named ‘sockets exhaustion’. For more information about this issue, see [You're using HttpClient wrong and it's destabilizing your software](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/) blog post.
+As a first issue, while this class implements `IDisposable`, using it with the `using` statement is not the best choice because when you dispose an `HttpClient` object, the underlying socket is not immediately released and this can cause a serious issue named ‘sockets exhaustion’. For more information about this issue, see the [You're using HttpClient wrong and it's destabilizing your software](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/) blog post.
```

Comment:
These changes didn't go in?

Reply:
Nope, they didn't.

I still don't get it 🙁. 

I work from the changed files tabs, because that's the only way to batch updates, but it seems like there are different sets of suggestions.

Creating another commit...